pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,21 +23,24 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+// Parallelism is used the store value from scheduler config
+var Parallelism = 16
```

Comment:
add int32 to make it explicitly an int32. are we setting `= 16` here for tests and other places that use parallelism without going through the whole scheduler bootstrap? defaults.go already sets a default, so I'm slightly hesitant to have the same number in two different places.

Reply:
We have added it here as the generic scheduler does not go through the bootstrapping

Reply:
Also why do we want to set `int32` specifically when the `n/Parallelism + 1; s > r ` computation requires an `int` ?

Reply:
I actually prefer just `int`. `int32` is only relevant in API configuration.

Also, note that other binaries (cluster autoscaler for example) don't bootstrap a scheduler, only a framework. I think this is fine for now.